### PR TITLE
string_serializations for #13

### PR DESCRIPTION
**Pull request by turbomam**
string_serializations for #13

not patterns

### CODE REVIEW

File: gsctools/mixs_converter.py [hunk 1]
```diff
@@ -251,6 +252,7 @@ def create_slot(self, row, enums: dict = {}) -> (str, Dict):
             if len(vals) > 2:
                 enum_name = f'{s_id}_enum'
                 slot['range'] = enum_name
+                slot['string_serialization'] = ''
```
Comment (cmungall) [inline gsctools/mixs_converter.py:255]: do you not want to do

```python
del slot['string_serialization']
```

**Review by cmungall — CHANGES_REQUESTED** [review]

File: gsctools/mixs_converter.py (hunk 1 above)
Comment (turbomam) [reply gsctools/mixs_converter.py:255]: will do

**Review by cmungall — APPROVED** [review]
I might be inclined to be more defensive with the deletion (test if present first), but go ahead and merge

**Comment by turbomam** [comment]
> I might be inclined to be more defensive with the deletion (test if present first), but go ahead and merge

thanks. created new issue.

**Pull request merged**